circles.js

```diff
@@ -0,0 +1,42 @@
+const gameContainer = document.createElement('div')
+gameContainer.classList.add('game-container')
+const circle = document.createElement('div')
+circle.classList.add('game-container__circle')
+let circles = Array.from({length: 5})
+circles.fill(circle)
+
+
+const createCircles = () => {
+    let id = 0
+    circles.forEach(() => {
+        const newCircle = circle.cloneNode(true)
+        newCircle.setAttribute('id', `${id}`)
+        id++
```

Comment:
bad mutation, why can't you use `index`?

Reply:
true